config/config.yaml

```diff
@@ -147,3 +147,8 @@ allowed_access_key_id_prefixes:
 
 # Allows to use slicer for Object uploading.
 internal_slicer: false
+
+# Requests
+
+# Maximum number of objects to be deleted per request limit by this value.
+max_object_to_delete_per_request: 1000
```

Comment:
Don't we have some section for S3-specific limits? Having this top-level doesn't seem to be nice and we probably have similar resource limits anyway.

Reply:
No, nothing appropriate for this. Separate it in `s3` section